sdk/core/azure-core-perf/src/main/java/com/azure/core/perf/PipelineSendTest.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.perf;
+
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.perf.core.CorePerfStressOptions;
+import com.azure.core.perf.core.RestProxyTestBase;
+import com.azure.core.util.BinaryData;
+import reactor.core.publisher.Mono;
+
+import java.io.UncheckedIOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.function.Supplier;
+
+public class PipelineSendTest extends RestProxyTestBase<CorePerfStressOptions> {
+
+    private final Supplier<BinaryData> binaryDataSupplier;
+    private final URL targetURL;
+
+    public PipelineSendTest(CorePerfStressOptions options) {
+        super(options);
+        binaryDataSupplier = createBinaryDataSupplier(options);
+        try {
+            targetURL = new URL(new URL(endpoint), "BinaryData");
+        } catch (MalformedURLException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    @Override
+    public void run() {
+        runAsync().block();
```

Comment:
shouldn't his be send.synchronously ?

Reply:
This is targeting `main` branch. I'll be changing this in sync stack branch when I merge main there. This in main is going to be useful to get comparison between sync and sync-over-async.